package.json

```diff
@@ -17,6 +17,7 @@
     "watch:test:unit": "nodemon --exec \"yarn test:unit\" ./test ./app ./ui",
     "sendwithprivatedapp": "node development/static-server.js test/e2e/send-eth-with-private-key-test --port 8080",
     "test:unit": "mocha --exit --require test/env.js --require test/setup.js --recursive \"test/unit/**/*.js\" \"ui/app/**/*.test.js\"",
+    "test:unit:permissions": "mocha --exit --require test/env.js --require test/setup.js --recursive \"test/unit/**/permissions/*.js\"",
```

Comment:
Why create a separate script for these tests?

Reply:
Ah, I was thinking about adding coverage requirements for the permissions controller. Thoughts on that?

Reply:
Oh I see! Yeah that could be worthwhile.

I wonder if there's a way to add coverage requirements without making it a separate script 🤔 Ah well - no big deal if not.

Reply:
Added coverage requirements in 374b88a

Reply:
For the record, I couldn't find a way to configure our existing script to require coverage for certain paths only: https://github.com/istanbuljs/nyc#selecting-files-for-coverage

Reply:
Good to know - thanks for looking into that